src/main.rs

```diff
@@ -17,6 +17,15 @@ const COMMANDS: &[(&str, (&str, &str))] = &[
         "test_rust",
         ("Run all Rust tests", "sh ./automation/all_rust_tests.sh"),
     ),
+    (
+        "verify_env",
+        (
+            "Verify your development environment",
+            "sh ./libs/verify-desktop-environment.sh ;
+            sh ./libs/verify-android-environment.sh ;
+            sh ./verify-ios-environment.sh",
```

Comment:
We can probably remove the extra options in the dialog:

![image](https://user-images.githubusercontent.com/128755/84678837-7e2c9a80-aefe-11ea-9e0b-ea80ba9af21c.png)